### PR TITLE
Joy

### PR DESCRIPTION
**Pull request by joyjoie**
#### What does this PR do?
It is for amending changes of Navigation Bar.
#### Description of Task completed
The design of the navbar has been modified and updated using HTML and Scss
####How should this be manually tested?
Use Yarn Build to run the application
#### Any background context you want to provide?
Use of Semantics UI for designing the Navbar
####What are the relevant pivotal tracker stories?
Presence of conflict in branches which were resolved by rebasing
#### Screenshots
Screenshot from 2019-09-20 12-32-55
#### Questions
Not at the moment

### CODE REVIEW

**Comment by Nicanor008** [comment]
- Squash your commits into one
- Update the PR description as below
Screenshot 2019-09-19 at 18 08 54
For more on PR description, check here https://github.com/ReachNet-LLC/Mooovers/wiki/Coding-Conventions

File: src/components/MenuLayout.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <Menu.Item as='a' header>
+          <Image src={logo} style={{ marginRight: '1.5em' }} />
+          
+        </Menu.Item>
+        
+        <Menu.Item as='a' style={{ paddingLeft: '40em' }}>Services</Menu.Item>
+        <Menu.Item as='a' >Contact Us</Menu.Item>
+        <Menu.Item as='a'>SignIn</Menu.Item>
+        <Menu.Item as='a' >Login</Menu.Item>
+
```
Comment (Nicanor008) [inline src/components/MenuLayout.js]: remove the whitespace

File: src/components/MenuLayout.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  Menu,
+ 
+} from 'semantic-ui-react'
+
+const MenuLayout = () => (
+  <div >
+    <Menu fixed='top' inverted>
+      <Container>
+        <Menu.Item as='a' header>
+          <Image src={logo} style={{ marginRight: '1.5em' }} />
```
Comment (Nicanor008) [inline src/components/MenuLayout.js]: css to be on external file

File: src/components/MenuLayout.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        
+        
+      </Container>
+    </Menu>
+
+  
+  </div>
+)
+
+export default MenuLayout
```
Comment (Nicanor008) [inline src/components/MenuLayout.js]: add an empty blank line

File: src/components/MenuLayout.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.navbar {
+	position: absolute;
+	width: 951px;
+	height: 54px;
+	left: -1px;
+	top: 0px;
+
+	background: #796FF0;
```
Comment (Nicanor008) [inline src/components/MenuLayout.scss]: i think this styling `background` is no longer being used, remove it

File: src/components/MenuLayout.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    background: #796FF0;
+    -webkit-box-shadow: none;
+    box-shadow: none;
+    color: #000000;
+}
+.ui.inverted.segment, .ui.primary.inverted.segment {
+    background: #796FF0;
+    color: rgba(255,255,255,.9);
+}
+.navbar{
```
Comment (Nicanor008) [inline src/components/MenuLayout.scss]: two `.navbar` on the same file, use one

File: src/routes/index.js [hunk 6]
```diff
@@ -1,9 +1,10 @@
 import React from "react";
 import { Switch, Route } from "react-router-dom";
 import HomePage from "../components/HomePage";
-
+import MenuLayout from "../components/MenuLayout";
```
Comment (Nicanor008) [inline src/routes/index.js]: change the import to `import MenuLayout from "../containers/MenuLayout";`

Comment (joyjoie) [reply src/routes/index.js]: I get a module error when i do this

File: src/components/MenuLayout.scss (hunk 5 above)
Comment (joyjoie) [reply src/components/MenuLayout.scss]: rectified

File: src/components/MenuLayout.scss (hunk 4 above)
Comment (joyjoie) [reply src/components/MenuLayout.scss]: it has been removed

File: src/components/MenuLayout.js (hunk 3 above)
Comment (joyjoie) [reply src/components/MenuLayout.js]: empty line before the export?

File: src/components/MenuLayout.js (hunk 2 above)
Comment (joyjoie) [reply src/components/MenuLayout.js]: Rectified

File: src/components/MenuLayout.js (hunk 1 above)
Comment (joyjoie) [reply src/components/MenuLayout.js]: Done

File: README.md [hunk 7]
```diff
@@ -1,26 +0,0 @@
-## Mooovers Web App
-Helps clients who want to move items from one place to another and also transport business who need customers.
-
-## UI Desgn
-Head over to [figma](https://www.figma.com/proto/znylbVjbVub4x91WxEGmyk/Cleaning-Services?node-id=7%3A27&viewport=393%2C-467%2C0.9057742357254028&scaling=scale-down)
-
-## Getting Started
-- Take to local computer `git clone https://github.com/ReachNet-LLC/Mooovers.git`
```
Comment (Nicanor008) [inline README.md]: Restore this file or rebase develop, it should not be deleted

**Comment by Nicanor008** [comment]
i can't see the logo, did you add it? and if so, did it show on your side?

**Comment by Nicanor008** [comment]
This should be the same as what is in the mockup, following that criteria, kindly check it
Screenshot 2019-09-19 at 18 32 52

File: src/components/MenuLayout.js (hunk 3 above)
Comment (Nicanor008) [reply src/components/MenuLayout.js]: empty line after `export default MenuLayout`

File: src/components/MenuLayout.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+const MenuLayout = () => (
+  <div>
+    <Menu fixed='top' inverted>
+      <Container>
+        <Menu.Item as='a' header>
+          <Image src={logo1} id="img1"/>
+          <Image src={logo2} id="img2"/>
+          <Image src={logo} id="img"/>
+
```
Comment (Nicanor008) [inline src/components/MenuLayout.js]: still there's this blank lines

File: src/containers/HomePage/index.js [hunk 9]
```diff
@@ -1,12 +1,21 @@
 import React, { Component } from 'react'
 import HomePage from '../../components/HomePage'
+<<<<<<< HEAD
```
Comment (Nicanor008) [inline src/containers/HomePage/index.js]: delete this header things and `=====` and `>>>>>mmmm`

**Pull request merged**